src/apiCalls.test.js

```diff
@@ -45,16 +61,16 @@ describe('apiCalls', () => {
     it('should return an error for response that is not ok', () => {
       window.fetch = jest.fn().mockImplementation(() => {
         return Promise.resolve({
-          ok: false,
-        })
-      })
+          ok: false
+        });
+      });
       expect(fetchAllMovies()).rejects.toEqual(Error('Error with GET request'));
     });
-  })
+  });
 
   describe('fetchUserLogin', () => {
-    let mockResponse = {id: 9, name: "Marge", email: "marge@turing.io"};
-    let mockEmail = "marge@turing.io";
+    let mockResponse = { id: 9, name: 'Marge', email: 'marge@turing.io' };
```

Comment:
Thank you for catching these linter inconsistencies.  You have a great eye for that!